housekeeper/store/api/base.py

```diff
@@ -1,30 +1,32 @@
 """This module defines a BaseHandler class holding different models"""
 
 from typing import Type
-from housekeeper.store.models import Bundle, File, Version, Tag
-from alchy import ModelBase, Query
+from housekeeper.store.models import Bundle, File, Version, Tag, Model
+from sqlalchemy.orm import Query, Session
 
 
 class BaseHandler:
     """This is a base class holding different models"""
 
-    Bundle: Type[ModelBase] = Bundle
-    Version: Type[ModelBase] = Version
-    File: Type[ModelBase] = File
-    Tag: Type[ModelBase] = Tag
+    Bundle: Type[Model] = Bundle
+    Version: Type[Model] = Version
+    File: Type[Model] = File
+    Tag: Type[Model] = Tag
```

Comment:
Do we need this anymore? I'm not sure why it was there in the first place?

Reply:
Hm, me neither! Good catch. But the tests fail when I remove them, seems like it requires some more refactoring! I'll do it in a separate PR